compile.sh

```diff
@@ -11,7 +11,7 @@ else
   echo 'Building spec'
   make -C spec
```

Comment:
May as well do a similar thing here and specify what outputs we actually want:
```suggestion
  make -C spec index.html webgpu.idl
```
even though it's the same as what's in the Makefile's default target currently.

Reply:
Thanks!